packages/radio-group/src/vaadin-radio-group.js

```diff
@@ -249,6 +249,23 @@ class RadioGroup extends FieldMixin(
     }
   }
 
+  /**
+   * Override an observer from `FieldMixin`.
+   *
+   * @param {boolean} invalid
+   * @protected
+   * @override
+   */
+  _invalidChanged(invalid) {
+    super._invalidChanged(invalid);
+
+    if (invalid) {
+      this.setAttribute('aria-invalid', 'true');
```

Comment:
I would suggest we also add a unit test as the `false` branch is currently not tested at all.